src/components/pages/MissionControl/RenderMissionControl.js

```diff
@@ -49,23 +49,23 @@ const RenderMissionControl = props => {
     return `checked=${e.target.checked}`;
   }
 
-  // redirects the user to the pdf of the story
+  // directs user to go in order; Read/Draw/Write
   const handleReadStory = e => {
     e.stopPropagation();
     push('/child/story');
   };
-  const handleWrite = e => {
-    e.stopPropagation();
-    if (!hasWritten && hasRead) {
-      push('/child/writing-sub');
-    }
-  };
   const handleDraw = e => {
     e.stopPropagation();
     if (!hasDrawn && hasRead) {
       push('/child/drawing-sub');
     }
   };
+  const handleWrite = e => {
+    e.stopPropagation();
+    if (!hasWritten && hasRead && hasDrawn) {
+      push('/child/writing-sub');
+    }
+  };
```

Comment:
def huge user flow error... will help make game play more intuitive and create less working errors

Reply:
I agree. This flow is much better for gameplay and will prevent users misunderstanding the drawing portion.